packages/create/src/cli/generate/adaptation-project.ts

```diff
@@ -78,7 +78,7 @@ async function generateAdaptationProject(
         if (!basePath) {
             basePath = join(process.cwd(), config.app.id);
         }
-        const fs = await generate(basePath, config);
+        const fs = await generate(basePath, ProjectType.ON_PREM, config);
```

Comment:
I think this config is missing some properties, when you pass the project type to the method you might see ts errors.

```suggestion
        const fs = await generate<ProjectType.ON_PREM>(basePath, ProjectType.ON_PREM, config);
```